crmsh/constants.py

```diff
@@ -230,6 +230,8 @@
     "timeout", "timestamp-format"
 )
 trace_ra_attr = "trace_ra"
+trace_dir_attr = "trace_dir"
+trace_dir_default = "/var/lib/heartbeat/trace_ra/Dummy/"
```

Comment:
I think we'd still like resource-agents to decide the default. So we could basically omit it in the cib if it's not explicitly specified by the user.